Tests/Reqnroll.RuntimeTests/BoDi/RegisterTypeTests.cs

```diff
@@ -65,7 +65,8 @@ public void ShouldNotAllowOverrideRegistrationAfterResolve()
 
             // when 
             Action act = () => container.RegisterTypeAs<SimpleClassWithDefaultCtor, IInterface1>();
-            act.Should().ThrowExactly<ObjectContainerException>();
+            var e = act.Should().ThrowExactly<ObjectContainerException>();
+            e.Which.Message.Should().Contain("IInterface1");
```

Comment:
There's a `.And` extension that lets you chain these assertions:

```suggestion
            act.Should().ThrowExactly<ObjectContainerException>()
                .And.Message.Should().Contain("IInterface1");
```